### ISSUE TITLE
Feat Req: Configurable intervals

### ISSUE THREAD

**Issue by nkfs** [outadoc/immich-home-assistant#5]
Hi,

Thanks for creating this project. I was wondering if the interval can be configured to something more frequent than 5m. Is it possible to be done in one of the scripts, or perhaps in the future within the UI?

Thanks

**Comment by janstadt** [comment]
Any chance we can get this feature added?

**Comment by joelis10** [comment]
+1 for wanting this feature, if not fully customisable I'd at least like options for 30s, 1m, 2m, 5m and 10m

**Comment by janstadt** [comment]
@outadoc Would you be able to get this working? Would really love to get this implemented so i can ditch the google photos integration. Just need this and crop mode seen here: https://github.com/outadoc/immich-home-assistant/issues/7

**Comment by janstadt** [comment]
another alternative to a configurable interval would be a service for skipping to the next image, that way we keep it at 5min and if someone wants to click to see a new image it'll just pull the new image.

**Comment by janstadt** [comment]
Im curious if this project is abandoned?

**Comment by joelis10** [comment]
I think it has, I'd recommend looking at this one that has many more features: https://github.com/damongolding/immich-kiosk

**Comment by janstadt** [comment]
Yeah i saw that one on reddit a few days ago. I would prefer having a native api and a card i think over standing up another docker container solely for an iframe embed. The device i'd be using it on is pretty low powered (Lenovo ThinkSmart) and i've found it doesnt handle iframes too great.

**Comment by Gamerayers** [comment]
Yes, I really like how this one functions, but want to define a much shorter interval than the 5 minutes. Make it configuratible.

**Comment by alpaclarry** [comment]
Sorry for grave digging but if anyone in the future comes across this thread you can edit the interval in custom_components\immich\image.py
I changed 
SCAN_INTERVAL = timedelta(minutes=5)
to 
SCAN_INTERVAL = timedelta(seconds=30) 
for a 30 second interal

**Comment by meks007** [comment]
switch to this fork: https://github.com/jianyu-li/immich-home-assistant

**Comment by jesserockz** [comment]
You can disable the polling and then force an update whenever you want using the action below
```yaml
action: homeassistant.update_entity
data:
  entity_id:
    - image.XXXX
```
